doc/developer/testing.md

```diff
@@ -224,7 +233,9 @@ provide most of the coverage, but it's worth adding some (*very*) basic
 testdrive tests (e.g., `> SELECT DATE '1999-01-01'`) to ensure that our pgwire
 implementation is properly serializing dates.
 
-To run a testdrive script, you'll need two terminal windows open. In the
+Like the [Unit Tests](#unitintegration-tests), in order to run testdrive, you should have Zookeeper, Kafka, and Confluent Schema Registry running. Testdrive is more flexible than the unit tests, though, in that you are allowed to run them at non-default locations.
```

Comment:
This link seems fishy?

Reply:
It works if you click it here: https://github.com/wangandi/materialize/blob/testnotes1/doc/developer/testing.md#unitintegration-tests

I just want the link to jump up to the "Unit Tests" section of the same document.

Reply:
Oh, I see; the section is "Unit/integration tests". Carry on.